lib/myxql/client.ex

```diff
@@ -43,8 +47,7 @@ defmodule MyXQL.Client do
         ssl_opts: Keyword.get(opts, :ssl_opts, []),
         connect_timeout: Keyword.get(opts, :connect_timeout, @default_timeout),
         handshake_timeout: Keyword.get(opts, :handshake_timeout, @default_timeout),
-        socket_options:
-          Keyword.merge([mode: :binary, packet: :raw, active: false], opts[:socket_options] || []),
+        socket_options: @sock_opts ++ (opts[:socket_options] || []),
```

Comment:
I think we want the opposite order because what comes first wins. Maybe we can also write a test? You can set `socket_options: [:inet]` for the test.

Reply:
Agreed, I will do that.